MMA Removal Utility/SingleTenantSetupInstallation.md

```diff
@@ -0,0 +1,302 @@
+# **Steps to install single-tenant AzTS MMA Removal Utility**
+
+In this section, we will walk you through the steps of setting up AzTS MMA Removal Utility. This setup can take up to 30 minutes.
+
+Before initiating the setup please validate the prerequisites, download and extract deployment package using [Prerequisites](./Prerequisites.md).
+
+There are two methods to setup AzTS MMA Removal Utility solution:
+
+- [**Method A:**](#method-a) This method provides granular level details of the different steps involved in setting up AzTS MMA Removal Utility. Setup process is divided into five steps and seprate command is provided for each step.
+
+- [**Method B:**](#method-b) Provides a quick way to install AzTS MMA Removal Utility solution by wrapping up multiple steps into a single consolidated command. 
+____________________________________________________________________
+
+## **Method A:**
+
+> _**Note:** Please validate the prerequisites [here](./Prerequisites.md). You can download the deployment package zip from [here](https://github.com/azsk/AzTS-docs/raw/main/TemplateFiles/AzTSMMARemovalUtilityDeploymentFiles.zip) and before extracting the zip file, right click on the zip file --> click on 'Properties' --> Under the General tab in the dialog box, select the 'Unblock' checkbox --> Click on 'OK' button. Extract the zip file and use **MMARemovalUtilitySetup.ps1** present in this package to run the commands mentioned in below steps_
+
+This setup is divided into five steps:
+
+1. [Load setup script](#step-1-of-5-load-setup-script)
+2. [Installing required Az modules](#step-2-of-5-installing-required-az-modules)
+3. [Setup remediation user-assigned managed identity](#step-3-of-5-setup-remediation-user-assigned-managed-identity)
+4. [Setup installation](#step-4-of-5-setup-installation)
+5. [Configure target scopes](#step-5-of-5-configure-target-scopes)
+
+Let's start!
+
+
+### **Step 1 of 5. Load setup script**
+ 
+ 1. Point current path to deployment folder and load AzTS MMA Removal Utility setup script <br/>
```

Comment:
Point the current path to the folder containing the downloaded deployment package and load the setup script for AzTS MMA Removal Utility

Reply:
Or did we intend to communicate something else?

Reply:
yes meant the same, incorporating the suggestion by tweaked the statement a bit to below.

Point the current path to the folder containing the extracted deployment package and load the setup script for AzTS MMA Removal Utility